@xen-orchestra/rest-api/src/xoa/xoa.service.mts

```diff
@@ -483,7 +483,7 @@ export class XoaService {
       return undefined
     })
     const nHostsEol = await this.#getNumberOfEolHosts().catch(err => {
-      log.err('#getNumberOfEolHosts failed', err)
+      log.error('#getNumberOfEolHosts failed', err)
```

Comment:
Already fixed in another PR:

[8695](https://github.com/vatesfr/xen-orchestra/pull/8695/files#diff-f6210a86b729af7331acb95819865eafb5353d6db1952698462da59aebb94b97R589)